apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -283,18 +299,10 @@ router.post('/*', (req, res, next) => {
         async () => {
           debug('Write edit to db');
           await createEdit(fileEdit);
-        },
-        (callback) => {
+
           debug('Write edit to disk (also push and sync if necessary)');
           fileEdit.needToSync = path.extname(fileEdit.fileName) === '.json';
-          saveAndSync(fileEdit, res.locals, (err) => {
-            // If there is an error, log it and pass null to the callback.
-            if (ERR(err, (err) => logger.info(err))) {
-              callback(null);
-              return;
-            }
-            callback(null);
-          });
+          await saveAndSync(fileEdit, res.locals);
```

Comment:
I know that I'm not telling you anything new here, but passing `res.locals` around is just so horrible!

Reply:
Tell me about it! What's even in this object???

Reply:
😭 😭 😭